code/mob/living/critter/changeling_critters.dm

```diff
@@ -257,7 +257,6 @@
 	setup_healths()
 		add_hh_flesh(5, 1)
 		add_hh_flesh_burn(4, 1.25)
-		add_health_holder(/datum/healthHolder/toxin)
```

Comment:
this is handspiders though?

Reply:
Oh my god i didn't notice, I'll fix this when I'm home. Thank you for noticing

Reply:
Yup fixed that now, safe to merge i think.